content/intro-to-storybook/react/en/deploy.md

```diff
@@ -1,91 +1,162 @@
 
-Next click the GitHub button to connect Netlify to GitHub. This allows it to access our remote Taskbox repo.
+And that's what we're going to implement next.
 
-Now select the taskbox GitHub repo from the list of options.
+### Add a GitHub Action to deploy Storybook
 
-![Netlify connect to repo](/intro-to-storybook/netlify-account-picker.png)
+In the root folder of our project, create a new directory called `.github`. Then create another directory called `workflows`.
 
-Configure Netlify by highlighting which build command to run in its CI and which directory the static site is outputted in. For branch choose `master`. Directory is `storybook-static`. Build command use `yarn build-storybook`.
+Create a new file called `chromatic.yml` like the one below, don't forget to change `project-token` with the actual project token.
+
+```yaml
+# .github/workflows/chromatic.yml
+# name of our action
+name: 'Chromatic Deployment'
+# the event that will trigger the action
+on: push
+
+# what the action will do
+jobs:
+  test:
+    # the operating system it will run on
+    runs-on: ubuntu-latest
+    # the list of steps that the action will go through
+    steps:
+      - uses: actions/checkout@v1
+      - run: yarn
+      - run: yarn build-storybook
+      - uses: chromaui/action@v1
+        # options required to the GitHub chromatic action
+        with:
+          # our project token, to see how to obtain it
+          # refer to https://www.learnstorybook.com/intro-to-storybook/react/en/deploy/
+          projectToken: project-token
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # the name of the folder where Storybook will be exported into.
+          storybookBuildDir: storybook-static
+```
+
+<div class="aside"><p>For brevity purposes <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets">GitHub secrets</a> weren't mentioned. Secrets are secure environment variables provided by GitHub so that you don't need to hard code the <code>project-token</code>.</p></div>
+
+### Commit the action
+
+In the command line, issue the following command to add the changes that were done:
+
+```bash
+git add .
+```
+
+Then commit them by issuing:
+
+```bash
+git commit -m "GitHub action setup"
+```
+
+Finally push them to the remote repository with:
+
+```bash
+git push origin master
+```
 
-![Netlify settings](/intro-to-storybook/netlify-settings.png)
+Once you’ve set up the GitHub action. Your Storybook will be deployed to Chromatic whenever you push code. You can find all the published Storybook’s on your project’s build screen in Chromatic.
 
-<div class="aside"><p>Should your deployment fail with Netlify, add the <a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a> flag to your <code>build-storybook</code> script.</p></div>
+![Chromatic user dashboard](/chromatic/chromatic-user-dashboard.png)
```

Comment:
@domyen this one i'll leave to your discretion. We could probably make it a bit better.

Reply:
Done 👍